components/count-badge/count-badge-icon.js

```diff
@@ -79,18 +91,31 @@ class CountBadgeIcon extends CountBadgeMixin(LitElement) {
 	}
 
 	render() {
-		const numberPadding = this.size === 'small' ? '0.55rem' : '0.7rem';
+		const numberPadding = this.size === 'small' ? '-0.65rem' : '-0.7rem';
 		const numberStyles = {
 			border: '2px solid white',
 			position: 'relative',
 			left: this.dir === 'rtl' ? 0 : 'var(--d2l-count-badge-icon-padding)',
 			right: this.dir === 'rtl' ? 'var(--d2l-count-badge-icon-padding)' : 0,
 			top: numberPadding,
-			visibility: this.skeleton ? 'hidden' : undefined
+			visibility: this.skeleton ? 'hidden' : undefined,
+			display: 'inline-block'
 		};
+
+		let horizontalPadding = '0.5rem';
+		const excessDigits = this.getNumberString().length - 4;
+		// large number strings could overflow onto the next badge,
+		// update padding in this case
+		if (excessDigits >= 0) {
```

Comment:
This part is a bit funky... looking for feedback here. 

For languages with long plus strings (ie. Plus de 99 in french), the bubble will overflow wayyy too far and hit other badges in the MCW. 

One option is to cap the width to prevent all overflows and have a vertical stack style. Pro: this will preserve the centering. Cons: looks weird and makes the badge quite tall. 
![multiword-stacked](https://user-images.githubusercontent.com/83968927/140997277-09cdbd36-6587-4581-8108-beb70c07392a.PNG)

The alternate approach I've gone with here is a best-of-both-worlds? For short strings less than 4 characters, allow a small amount of overflow. This will never get to other badges. This will preserve the centering. 
For long strings, we know it will absolutely overflow. We can add some backup padding in this case. Here, I've got 0.5 for each extra character over 4 characters. The centering will not be preserved in this case, but at least we will be able to read the number. 
![extra-padding-long-nums-only](https://user-images.githubusercontent.com/83968927/140997530-1673716e-61b4-45ac-a401-3320f9f6d70e.PNG)

Not sure which approach to go with. We can also scrap the long langterms and go back to 99+. This will still allow overflow for count badges without limits, like in Portfolio.

Reply:
I was away when these translations came back, but now that the implication of their choice to not just stick with `99+` is even larger, should we check again that it's actually the correct interpretation for those languages? i.e. could we make these not language terms at all and just use `99+` for ALL languages.

Reply:
Yep that was my thought too. I'll mention it in standup tomorrow. I think it's still good to have the padding function as a backup for the case where we have a super long number. I'm not sure it should be up to the consumers to be monitoring the size of the bubble which would require going into the shadowroot.

Reply:
So if the positioning is done like my codepen below, it will "overflow" to the left overtop of the icon. Then maybe we could cap it at X characters such it can never be large enough to overflow beyond the left boundary of the icon.